3DAmsterdam/Assets/WebGLTemplates/Fullscreen3DAmsterdam/index.html

```diff
@@ -139,10 +122,10 @@
 		  //Reset file selection
 		  document.getElementById("obj").value = "";
 	  }
-	  function AbortInUnity() {
-		  console.log("Abort import in Unity");
-		  unityInstance.SendMessage(unityObjectName, "AbortImport");
-		  document.getElementById("obj").value = "";
```

Comment:
Hi bart, did you maybe merge the index.html from develop and replace it with your file? Because this method should not be removed, and I see some other changes that were undone in the html file

Reply:
ah, that could be true, will fix!

Reply:
changed it